control/statesp.py

```diff
@@ -611,6 +611,102 @@ def feedback(self, other=1, sign=-1):
 
         return StateSpace(A, B, C, D, dt)
 
+    def lft(self, other, nu=-1, ny=-1):
+        """Return the Linear Fractional Transformation.
+
+        See definition here:
+        https://www.mathworks.com/help/control/ref/lft.html
```

Comment:
Can we reference a textbook or some other more "definitive" source for the definition of LFT?